ls/server/src/main/java/org/opencds/cqf/cql/ls/server/command/CqlCommand.java

```diff
@@ -234,22 +231,24 @@ public Integer call() throws Exception {
         return 0;
     }
 
-    private Repository createRepository(FhirContext fhirContext, Path terminologyPath, Path modelPath) {
+    private Repository createRepository(FhirContext fhirContext, String terminologyUrl, String modelUrl) {
```

Comment:
Even though behavior around the terminology path is not affected by this change, I've made it a `String` as well for consistency. `createRepository` is now responsible for turning the given URLs, if present, into the appropriate form for a repository.